.packit.yml

```diff
@@ -45,6 +45,8 @@ jobs:
     trigger: pull_request
     identifier: integration-tests
     fmf_path: tests
+    labels:
+      - basic
```

Comment:
Not sure about the name, `basic` evokes only some minimal set of tests, how about `standard` or `classic`?

Reply:
Naming was an issue, I didn't like basic too much either
Standard is good